site/_wiki/Development/Configurations.md

```diff
@@ -81,6 +81,7 @@ pinpoint devices.
 | Output Initialization Report  |       `List<byte[]>`       | A list of output reports to be sent to the device to perform the device's initialization sequence.
 |        Device Strings         | `Dictionary<byte, string>` | A list of regular expressions to be matched against specific indexes of strings contained within the device's firmware. They can be retrieved via the device string reader. This is optional, however it is commonly used to improve detection precision.
 | Initialization String Indexes |          `byte[]`          | A list of indexes to be retrieved from the device as part of the device's initialization sequence. This is optional, and very infrequently used.
+|         `Interface`           |    non-negative integer    | Specifies the USB device interface to use. For example: `0`. Should be used when the identifier might be ambiguous, or if the tablet has multiple interfaces matching the used identifiers, but only has one of them usable.
```

Comment:
This is nested within `Attributes`, its not just sitting there raw within the identifier.

Additionally (though OTD accepts it, only the tests don't) it should be encoded as a string, so `"0"`, not `0`.

Reply:
Whoops, didn't notice the column header differed. And good catch on it being nested.

Reply:
I think this is resolved with 59b8d9c